app/views/content_providers/_content_provider.html.erb

```diff
@@ -10,7 +10,7 @@
       </div>
 
       <% if content_provider.materials.length > 0 %>
-        <div><strong><%= pluralize(content_provider.materials.length, 'training material') %></strong></div>
+        <div><strong><%= pluralize(content_provider.materials.not_disabled.length, 'training material') %></strong></div>
```

Comment:
Add a simple test for this that confirms that a disabled material is not included.